testcases.py

```diff
@@ -20,8 +20,8 @@
 KB = 1 << 10
 MB = 1 << 20
 
-QUIC_DRAFT = 29  # draft-29
-QUIC_VERSION = hex(0xFF000000 + QUIC_DRAFT)
+QUIC_DRAFT = 34  # draft-34
```

Comment:
```suggestion
```

Reply:
It's still used in https://github.com/marten-seemann/quic-interop-runner/blob/8a0ffcc8ae7d60f8e2698e8b08a78becc2589bee/interop.py#L246-L247

Reply:
We can remove it, but then need to give up on showing the draft version on the results page. Which we could do.